packages/es-components-via-theme/index.js

```diff
@@ -18,6 +18,7 @@ const blue100 = '#b3e3f2';
 const blue300 = '#4dbde0';
 const blue500 = '#00a0d2';
 const blue700 = '#008ec7';
+const blue800 = '#f2f8fb';
```

Comment:
I'd prefer this be called something else, since the numbers (300, 500, 700, etc.) are steps in a linear gradient. Putting this color in the middle of it breaks that. The color you're adding should be in the gray gradients because it's barely distinguishable from white. Maybe "offwhite" for the name?

Reply:
Also suggest either "eggshell" or a name that better describes what it's used for... "tileBackground1"? It does look a little more blue-y in the screenshot you provided, so maybe "blue000".

Reply:
Just wanna holler because I _really_ appreciate this context about the linear gradient steps!

Reply:
Ya thanks for the heads up.  I didn't know that was what it was for.

When you say `blue75`, you mean adding it here?
https://github.com/WTW-IM/es-components/blob/main/packages/es-components-via-theme/index.js#L17

Reply:
Yep, I think you could add it as blue75 there between blue50 and blue100 and that would make the most sense contextually.